src/components/Tables/AppletsTable/Table/AppletItem/AppletItem.tsx

```diff
@@ -69,11 +73,20 @@ export const AppletItem = ({ item }: { item: FolderApplet }) => {
           <Actions items={actionsRender(actions)} context={item} />
         </TableCell>
       </TableRow>
-      <DeletePopUp
-        deleteModalVisible={deleteModalVisible}
-        setDeleteModalVisible={setDeleteModalVisible}
-        item={item}
-      />
+      {deletePopupVisible && (
```

Comment:
Why here  {deletePopupVisible && ( statement

Reply:
why do we need to store the component and its state after closing?

Reply:
ok, make sense